doc/tutorial.rst

```diff
@@ -176,8 +177,8 @@ Let’s confirm this by comparing the prediction interval widths over
 .. code:: python
 
     fig, ax = plt.subplots(1, 1, figsize=(7, 5))
-    for strategy in strategies:
-        ax.plot(X_test, prediction_interval[strategy][:, 2] - prediction_interval[strategy][:, 1])
+    for strategy in STRATEGIES.keys():
```

Comment:
Tu n'es pas obligé de préciser `keys()`, un dictionnaire est itéré nativement sur ses clefs

Reply:
Done